crates/wit-component/dl/build.sh

```diff
@@ -7,4 +7,4 @@
 
 CARGO_PROFILE_RELEASE_LTO=true RUSTFLAGS="-C relocation-model=pic" cargo build --release --target=wasm32-wasi
 $WASI_SDK_PATH/bin/clang -shared -o $1 -Wl,--whole-archive ../../../target/wasm32-wasi/release/libdl.a -Wl,--no-whole-archive
-cargo run --manifest-path ../../../Cargo.toml -- strip -a $1 -o $1
+cargo run --manifest-path ../../../Cargo.toml -- strip $1 -o $1
```

Comment:
Optional idea: we could run `strip` a second time using `--delete name`, which would remove the `name` section but keep the others.

Reply:
Sounds reasonable to me 👍